react/features/subtitles/middleware.js

```diff
@@ -172,6 +176,11 @@ function _requestingSubtitlesToggled({ getState }) {
     const { _requestingSubtitles } = state['features/subtitles'];
     const { conference } = state['features/base/conference'];
 
+    sendAnalytics(createToolbarEvent('transcribing.ccButton',
```

Comment:
Is the purpose of this change to send the toolbar event when the transcription is turned on from the recording dialog?

If yes, isn't it better if we have another event for this or if we include it somehow in the recording event? I'm worried for the scenario when we want to check how many users are using the transcriber without the recording for example. WDYT?

Reply:
I see, I will check it, thanks.